src/HasTranslations.php

```diff
@@ -91,6 +91,18 @@ public function getTranslations($key) : array
         return json_decode($this->getAttributes()[$key] ?? '' ?: '{}', true) ?: [];
     }
 
+    public function getAllTranslations() : array
```

Comment:
Instead of adding a dedicated method, we'd could let this work by don't passing a `$key` in `getTranslations`.

Reply:
Thank you. I don't know why I didn't do it this way. Will revise.